src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/stats/MflStatSpec.groovy

```diff
@@ -353,4 +365,23 @@ class MflStatSpec extends HealthCheckSpecification {
             flowHelperV2.deleteFlow(flow.flowId)
         }
     }
+
+    def switchIsConnectedToFl(SwitchId switchId, Boolean mgmt, Boolean stats) {
```

Comment:
Stats region have read-only mode, you can relly on it in such checks.

Reply:
thx, refactored